.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -74,9 +57,20 @@ object CalypsoApps: BuildType({
 	name = "CalypsoApps"
```

Comment:
Nit: Maybe we could add a space, to change this to `Calypso Apps` so it's consistent with the other build names.

```suggestion
	name = "Calypso Apps"
```